dts/arm64/nxp/nxp_mimx8mn_a53.dtsi

```diff
@@ -143,4 +143,35 @@
 		compatible = "nxp,rdc";
 		reg = <0x303d0000 DT_SIZE_K(64)>;
 	};
+
+	enet: enet@30be0000 {
+		compatible = "nxp,enet1g";
+		reg = <0x30be0000 DT_SIZE_K(64)>;
+		clocks = <&ccm IMX_CCM_ENET_CLK 0 0>;
+		rdc = <RDC_DOMAIN_PERM(A53_DOMAIN_ID, RDC_DOMAIN_PERM_RW)>;
```

Comment:
I didn't notice before when we added this, but the description of the rdc property needs to be improved in the binding including instruction about this macro

Reply:
current descriptions is: Set the RDC(Resource Domain Controller) domain permission for this peripheral, so what's your suggestion?

Reply:
please update the binding with information about how to use this macro and all the options

Reply:
Good suggestion, I will create another PR for that as it is not introduced by this PR, thanks.

Reply:
Created PR https://github.com/zephyrproject-rtos/zephyr/pull/76764 to address this comment.